core/controller/src/main/scala/whisk/core/entitlement/Collection.scala

```diff
@@ -109,6 +110,10 @@ protected[core] object Collection {
 
   protected[core] def requiredProperties = WhiskEntityStore.requiredProperties
 
+  /** Number of records allowed per query. */
+  protected[core] val DEFAULT_LIST_LIMIT = 30
```

Comment:
Would be nice if `DEFAULT_LIST_LIMIT` and `MAX_LIST_LIMIT` were configurable with pureconfig, but we can do that later.